selfdrive/boardd/panda_comms.cc

```diff
@@ -94,16 +94,14 @@ void PandaUsbHandle::cleanup() {
 }
 
 std::vector<std::string> PandaUsbHandle::list() {
+  static std::unique_ptr<libusb_context, decltype(&libusb_exit)> context(init_usb_ctx(), libusb_exit);
```

Comment:
does this still handle new USB devices coming up after making the first context?

Reply:
I think it's okay.
https://github.com/libusb/libusb/blob/cc3df7760926bd98358e3d2c76c1511fe7861365/libusb/core.c#L822

Libusb will return the most recent listings if hotplug is supported. Otherwise it will redo the query.